src/main/java/com/epam/lowcost/model/Flight.java

```diff
@@ -23,6 +23,9 @@
     private String departureAirport;
     private String arrivalAirport;
     private boolean isDeleted;
+    private long placePriorityPrice;
+    private  long businessPrice;
```

Comment:
Do you see it too?)))) @OlgaKochneva  ?))

Reply:
Noooo, what should I see?

Reply:
Yesterday we told about code style and now again. It is obvious, common...

Reply:
corrected